src/routes/users.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const express = require('express');
+const userController = require('../controllers/users');
+const router = express.Router();
+
+router.get('/', userController.getAll);
+router.get('/:userId', userController.getOne);
```

Comment:
Consider using a more descriptive name for the route parameter, such as ':id' instead of ':userId'. This will make the code more readable and maintainable.